vb-fe/src/services/Room.ts

```diff
@@ -8,6 +8,10 @@ export const createRoom = (room: RoomForm, userId: string): Promise<any> => {
   return Http.post(`${RoomEndpoints.GENERAL}`, { room, userId });
 };
 
+export const updateRoom = (room: RoomForm, userId: string, roomId: string): Promise<any> => {
+  return Http.put(`${RoomEndpoints.GENERAL}${roomId}`, { room, userId });
```

Comment:
can you refactor the `RoomEndpoints.GENERAL` to be `api/room` (remove the trailing slash). None of the other endpoints have that. You'd have to change this too, to `${RoomEndpoints.GENERAL}/${roomId}`